src/tcgwars/logic/impl/gen3/CrystalGuardians.groovy

```diff
@@ -1904,7 +1936,7 @@ public enum CrystalGuardians implements LogicCardInfo {
           }
         }
         playRequirement{
-          assert all.findAll {it.cards.hasType(POKEMON_TOOL)} || (bg.stadiumInfoStruct && stadiumCanBeAffectedByItemAndSupporter())
+          assert all.any{it.cards.hasType(POKEMON_TOOL)} || (bg.stadiumInfoStruct && stadiumCanBeAffectedByItemAndSupporter())
```

Comment:
Is this trainer at all different than field blower? If not, one should probably copy the other.

Reply:
Looking at them, they should be functionally identical. Will use copy on this 👍